### PR TITLE
Development: updated info re updating Cypress snapshots tests

### PR DESCRIPTION
**Pull request by AlbertCarreras**
Development: updated info re updating Cypress snapshots tests [MAIN CHANGE](https://github.com/pinterest/gestalt/pull/1978#discussion_r822860106)

All other noisy changes are just renames of `screenshot` to `snapshot` references for consistency.

### CODE REVIEW

File: docs/pages/development.js [hunk 1]
```diff
@@ -243,6 +243,63 @@ yarn cypress open
 ~~~"
                   />
                 </li>
+                <li>
```
Comment (AlbertCarreras) [inline docs/pages/development.js:246]: MAIN CHANGE

File: docs/pages/development.js [hunk 2]
```diff
@@ -243,6 +243,63 @@ yarn cypress open
...
+                      <Image
+                        alt=""
+                        color="white"
+                        naturalHeight={262}
+                        naturalWidth={400}
+                        src="https://i.ibb.co/FY1pp4Y/Screen-Shot-2022-03-09-at-4-49-21-PM.png"
+                      />
+                      <Text size="100" align="center">
+                        <Box as="figcaption" marginTop={3}>
+                          {`Each failing snapshot test with display an error message. Click on
```
Comment (ayeshakmaz) [inline docs/pages/development.js:280]: typo: will display instead of with display

File: docs/pages/development.js [hunk 3]
```diff
@@ -243,6 +243,63 @@ yarn cypress open
...
+yarn start
+~~~"
+                  />
+                  <Markdown
+                    text="
+~~~bash
+yarn cypress open
+~~~"
+                  />
+                  <Flex direction="column">
```
Comment (ayeshakmaz) [inline docs/pages/development.js:269]: Could you try placing these side by side, and letting them wrap on small screens? Might help avoid large white space on wide screens

**Review by ayeshakmaz — APPROVED** [review]
One typo, one suggestion you can leave or take : )

File: cypress/integration/visual-test/ActivationCard-dark_spec.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ describe('ActivationCard dark mode visual regression check', () => {
     cy.visit('/visual-test/ActivationCard-dark');
   });
 
-  it('Compares screenshots', () => {
+  it('Compares snapshots', () => {
```
Comment (dangerismycat) [inline cypress/integration/visual-test/ActivationCard-dark_spec.js:6]: This PR would be a lot easier to review if these changes had been made in a separate PR

File: docs/pages/development.js [hunk 5]
```diff
@@ -243,6 +243,63 @@ yarn cypress open
 ~~~"
                   />
                 </li>
+                <li>
+                  <Text>
+                    Run{' '}
+                    <Link href="https://github.com/meinaart/cypress-plugin-snapshots" inline>
+                      <Text weight="bold">Cypress snapshot tests</Text>
```
Comment (dangerismycat) [inline docs/pages/development.js:250]: Given the use of "snapshot" in Jest, we should clarify that these are visual diff tests. Something like "Cypress visual diff snapshot tests"?

File: docs/pages/development.js [hunk 6]
```diff
@@ -243,6 +243,63 @@ yarn cypress open
...
+                    </Link>
+                    . If any component changes are expected to visually modify your component, you
+                    must update the snapshot tests. In order to run the tests, you must start the
+                    documentation server in one terminal and the Cypress testing interface in
+                    another.
+                  </Text>
+                  <Markdown
+                    text="
```
Comment (dangerismycat) [inline docs/pages/development.js:258]: consider labeling these something like `Terminal tab #1` and `Terminal tab #2` for clarity

File: docs/pages/development.js [hunk 7]
```diff
@@ -243,6 +243,63 @@ yarn cypress open
...
+                        alt=""
+                        color="white"
+                        naturalHeight={176}
+                        naturalWidth={400}
+                        src="https://i.ibb.co/7NyhPRD/Screen-Shot-2022-03-09-at-4-49-40-PM.png"
+                      />
+                      <Text size="100" align="center">
+                        <Box as="figcaption" marginTop={3}>
+                          {`On this new interface displaying the expected and the received snapshots,
+                          click on the "Update snapshot" button to reconcile the changes.`}
```
Comment (dangerismycat) [inline docs/pages/development.js:297]: This should include language about updating only if the changes are expected, otherwise to revisit the code

**Review by dangerismycat — APPROVED** [review]

File: docs/pages/development.js (hunk 2 above)
Comment (AlbertCarreras) [reply docs/pages/development.js:280]: DONE

File: cypress/integration/visual-test/ActivationCard-dark_spec.js (hunk 4 above)
Comment (AlbertCarreras) [reply cypress/integration/visual-test/ActivationCard-dark_spec.js:6]: I only did this cos main change was only in a single page, so everything else could be skipped.

File: docs/pages/development.js (hunk 5 above)
Comment (AlbertCarreras) [reply docs/pages/development.js:250]: DONE

File: docs/pages/development.js (hunk 6 above)
Comment (AlbertCarreras) [reply docs/pages/development.js:258]: DONE

File: docs/pages/development.js (hunk 3 above)
Comment (AlbertCarreras) [reply docs/pages/development.js:269]: DONE

File: docs/pages/development.js (hunk 7 above)
Comment (AlbertCarreras) [reply docs/pages/development.js:297]: DONE

**Pull request merged**